src/networking/VoteService.ts

```diff
@@ -26,6 +26,7 @@ export default class VoteService extends ModelService {
           WHERE_OPERATOR.eq,
           doc(NetworkService.getDb(), COLLECTION_NAME.Event, eventId)
         ),
+        orderBy(REF_PROPERTY_NAME.VotePersonRef),
```

Comment:
I assume this is to get the same ordering every time the page loads?

Reply:
Yes, I think so.